src/invariant/OrderBookIsNotCrossed.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "invariant/Invariant.h"
+#include "xdr/Stellar-ledger-entries.h"
+
+#include <unordered_map>
```

Comment:
I don't think you need this include

Reply:
I think I do. `Invariant.h` only includes:
```
#include <memory>
#include <string>
```